Sources/generate-swift-syntax-builder/gyb_syntax_support/AttributeKinds.py

```diff
@@ -0,0 +1,871 @@
+        "CONTEXTUAL_DECL_ATTR_ALIAS(" + self.name + ', ' + self.klass + ")"
+
+# Abstract class aggregations for use in Attr.def.
+OnValue = 'OnValue'
+OnTypeAlias = 'OnTypeAlias'
+OnEnumElement = 'OnEnumElement'
+OnSubscript = 'OnSubscript'
+OnVar = 'OnVar'
+OnExtension = 'OnExtension'
+OnClass = 'OnClass'
+OnFunc = 'OnFunc'
+OnAccessor = 'OnAccessor'
+OnEnum = 'OnEnum'
+OnConstructor = 'OnConstructor'
+OnStruct = 'OnStruct'
+OnImport = 'OnImport'
+OnAssociatedType = 'OnAssociatedType'
+OnGenericTypeParam = 'OnGenericTypeParam'
+OnParam = 'OnParam'
+OnNominalType = 'OnNominalType'
+OnProtocol = 'OnProtocol'
+OnConcreteNominalType = 'OnConcreteNominalType'
+OnGenericType = 'OnGenericType'
+OnAbstractFunction = 'OnAbstractFunction'
+OnOperator = 'OnOperator'
+OnAnyDecl = 'OnAnyDecl'
+
+# True if multiple instances of this attribute are allowed on a single
+# declaration.
+AllowMultipleAttributes = 'AllowMultipleAttributes'
+
+# True if this is a decl modifier - i.e., that it should not be spelled
+# with an @.
+DeclModifier = 'DeclModifier'
+
+# True if this is a long attribute that should be printed on its own line.
+#
+# Currently has no effect on DeclModifier attributes.
+LongAttribute = 'LongAttribute'
+
+# True if this shouldn't be serialized.
+NotSerialized = 'NotSerialized'
+
+# True if this attribute is only valid when parsing a .sil file.
+SILOnly = 'SILOnly'
+
+# The attribute should be reported by parser as unknown.
+RejectByParser = 'RejectByParser'
+
+# Whether client code cannot use the attribute.
+UserInaccessible = 'UserInaccessible'
+
+# Whether adding this attribute can break API
+APIBreakingToAdd = 'APIBreakingToAdd'
+
+# Whether removing this attribute can break API
+APIBreakingToRemove = 'APIBreakingToRemove'
+
+# Whether adding this attribute can break ABI
+ABIBreakingToAdd = 'ABIBreakingToAdd'
+
+# Whether removing this attribute can break ABI
+ABIBreakingToRemove = 'ABIBreakingToRemove'
+
+# The opposite of APIBreakingToAdd
+APIStableToAdd = 'APIStableToAdd'
+
+# The opposite of APIBreakingToRemove
+APIStableToRemove = 'APIStableToRemove'
+
+# The opposite of ABIBreakingToAdd
+ABIStableToAdd = 'ABIStableToAdd'
+
+# The opposite of ABIBreakingToRemove
+ABIStableToRemove = 'ABIStableToRemove'
+
+# Whether this attribute is only valid when concurrency is enabled.
+ConcurrencyOnly = 'ConcurrencyOnly'
+
+# Whether this attribute is valid on additional decls in ClangImporter.
+OnAnyClangDecl = 'OnAnyClangDecl'
+
+TYPE_ATTR_KINDS = [
+  # Type attributes
+  TypeAttribute('autoclosure'),
+  TypeAttribute('convention'),
+  TypeAttribute('noescape'),
+  TypeAttribute('escaping'),
+  TypeAttribute('differentiable'),
+  TypeAttribute('noDerivative'),
+  TypeAttribute('async'),
+  TypeAttribute('Sendable'),
+  TypeAttribute('unchecked'),
+  TypeAttribute('_typeSequence'),
+  TypeAttribute('_local'),
+  TypeAttribute('tuple'),
+
+  # Generated interface attributes
+  TypeAttribute('_opaqueReturnTypeOf'),
+]
+
+# Schema for DECL_ATTR:
```

Comment:
This comment still references the C++ macro names (`DECL_ATTR` etc.). It should probably use the Python names (`DeclAttr` etc.)